src/sage/coding/reed_muller_code.py

```diff
@@ -401,6 +401,14 @@ class directly, as :meth:`ReedMullerCode` creates either a binary or a
         sage: C = codes.BinaryReedMullerCode(2, 4)
         sage: C
         Binary Reed-Muller Code of order 2 and number of variables 4
+
+    Very large Reed-Muller codes can be constructed without building the generator matrix or elements of the code.
```

Comment:
```diff
-    Very large Reed-Muller codes can be constructed without building the generator matrix or elements of the code.
+    Very large Reed-Muller codes can be constructed without building
+    the generator matrix or elements of the code::
```

Reply:
Also refer to the issue number `:issue:\`39110\``.

Reply:
Thanks for your reviews. I fixed the doc.